libs/vkd3d/swapchain.c

```diff
@@ -1586,6 +1731,13 @@ static void dxgi_vk_swap_chain_recreate_swapchain_in_present_task(struct dxgi_vk
 
     INFO("Got %u swapchain images.\n", chain->present.backbuffer_count);
 
+    /* If low latency is supported restore the current low latency state now */
+    if (chain->queue->device->vk_info.NV_low_latency2)
```

Comment:
Can release lock and set mode before querying images.

Reply:
Done.